prog/vm/host_nexus.rb

```diff
@@ -117,11 +117,22 @@ def before_run
       spdk_installation = vm_host.spdk_installations.first
       spdk_cores = (spdk_installation.cpu_count * vm_host.total_cores) / vm_host.total_cpus
       vm_host.update(used_cores: spdk_cores)
-      hop_prep_reboot
+      hop_download_boot_image
     end
     donate
   end
 
+  label def download_boot_image
+    bud Prog::DownloadBootImage, {"image_name" => Config.default_boot_image_name, "version" => Config.default_boot_image_version}
```

Comment:
What if we dynamically determine the images needed to download to the host? It's just an idea.

An example:

```ruby
def default_boot_images
    return [] if Config.development?
    # Should we download runner images to the all hosts?
    images = ["ubuntu-jammy", "github-ubuntu-2204", "github-ubuntu-2004"]
    
    # Germany hosts should have PG image too
    images << "postgres-ubuntu-2204" if vm_host.location == "hetzner-fsn1"

    # GPU hosts should have runner image too
    images << "github-gpu-ubuntu-2204" if vm_host.pci_devices_dataset.count > 0

    images
end
```

Reply:
Makes sense, but this requires https://github.com/ubicloud/ubicloud/pull/1570 to be merged first.

Also, I think we should have `ubuntu-jammy` for development environment.